tools/depends/target/libspdlog/Makefile

```diff
@@ -12,7 +12,8 @@ CMAKE_OPTIONS= \
 	-DSPDLOG_BUILD_TESTS=OFF \
 	-DSPDLOG_BUILD_BENCH=OFF \
 	-DSPDLOG_INSTALL=ON \
-	-DSPDLOG_FMT_EXTERNAL=ON
+	-DSPDLOG_FMT_EXTERNAL=ON \
+	-DCMAKE_PREFIX_PATH=${PREFIX}
```

Comment:
I'm not sure if it's a good idea to pass this for full depends builds too. I would add it to the CMAKE variable in line 31.

Reply:
You're right I actually wanted to do that but must have ended up at the wrong place.

Reply:
Fixed